appIos/post/post/PostScreen.swift

```diff
@@ -179,7 +183,7 @@ private struct PostAuthor: View {
                     font: theme.typography.subtitle
                 )
                 AppText(
-                    text: "1 min read",
+                    text: String(format: "%d min read", readingTimeMinutes),
```

Comment:
I haven't checked the equivalent to the `strings` resources in Android. I guess that's the proper way to format this.

Reply:
Nah it’s okay for now - there is a way but the project doesn’t support yet which is a separate piece of work.